src/test/java/subway/LineAcceptanceTest.java

```diff
@@ -0,0 +1,190 @@
+            .when()
+            .post("/lines")
+            .then()
+            .log().all()
+            .statusCode(HttpStatus.CREATED.value())
+            .extract();
+    }
+
+    private ExtractableResponse<Response> getSubwayLines() {
+        return given()
+            .accept(MediaType.APPLICATION_JSON_VALUE)
+            .when().get("/lines")
+            .then()
+            .statusCode(HttpStatus.OK.value())
+            .extract();
+    }
+
+    /**
+     * 		given 2개의 지하철노선을 생성하고
+     * 	  when 지하철노선 목록을 조회하면
+     * 	  then 지하철 노선 목록 조회 시 생성한 노선을 찾을 수 있다
+     */
+
+    @DisplayName("지하철 노선 목록을 조회")
+    @Test
+    void 지하철_노선_목록_조회() {
+        // given
+        createSubwayLine(LINE_SHIN_BUN_DANG_REQUEST);
+        createSubwayLine(LINE_TWO_REQUEST);
+
+        // when
+        ExtractableResponse<Response> response = getSubwayLines();
+
+        // then
+        List<String> lineNames = response.jsonPath().getList("name", String.class);
+        assertThat(lineNames).containsAnyOf(LINE_SHIN_BUN_DANG_REQUEST.getName(),
+            LINE_TWO_REQUEST.getName());
+    }
+
+    private ExtractableResponse<Response> getSubwayLine(Long id) {
+        return given()
+            .accept(MediaType.APPLICATION_JSON_VALUE)
+            .pathParam("id", id)
+            .when().get("/lines/{id}")
+            .then()
+            .statusCode(HttpStatus.OK.value())
+            .extract();
+    }
+
+
+    /**
+     * Given 지하철 노선을 생성하고
+     * When 생성한 지하철 노선을 조회하면
+     * Then 생성한 지하철 노선의 정보를 응답받을 수 있다.
+     */
+
+    @DisplayName("지하철노선 조회")
+    @Test
+    void 지하철노선_조회() {
+        //given
+        ExtractableResponse<Response> createdResponse = createSubwayLine(
+            LINE_SHIN_BUN_DANG_REQUEST);
+
+        //when
+        Long id = createdResponse.jsonPath().getLong("id");
+        ExtractableResponse<Response> getResponse = getSubwayLine(id);
+
+        //then
+        String name = getResponse.jsonPath().getString("name");
+        assertThat(name).isEqualTo(LINE_SHIN_BUN_DANG_REQUEST.getName());
+
+    }
+
+    private void updateSubwayLine(Long id) {
+        RestAssured.given()
+            .pathParam("id", id)
+            .log().all()
+            .body(LineRequest.of(SubwayFixture.LINE_NEW_SHIN_BUN_DANG_REQUEST.getId(),
+                SubwayFixture.LINE_NEW_SHIN_BUN_DANG_REQUEST.getName(),
+                SubwayFixture.LINE_NEW_SHIN_BUN_DANG_REQUEST.getColor()))
+            .contentType(MediaType.APPLICATION_JSON_VALUE)
+            .when().put("/lines/{id}")
+            .then().statusCode(HttpStatus.OK.value())
+            .log().all()
+            .extract();
+    }
+
+    /**
+     * Given 지하철 노선을 생성하고
+     * When 생성한 지하철 노선을 수정하면
+     * Then 해당 지하철 노선 정보는 수정된다
+     */
+    @DisplayName("지하철노선 수정")
+    @Test
+    void 지하철노선_수정() {
+        //given
+        ExtractableResponse<Response> createdResponse = createSubwayLine(
+            LINE_SHIN_BUN_DANG_REQUEST);
```

Comment:
```suggestion
        Long createdResponse = createSubwayLine(
            LINE_SHIN_BUN_DANG_REQUEST);
```

Response가 아닌, Id를 반환하면 더 간결해지지 않을까요~?